### PR TITLE
Adding view files, updating pre existing files, and adding images

### PR DESCRIPTION
**Pull request by GooBall**
Added all the currently used 'view' files along with some of their dependencies, in addition to all the image files.

I've updated most of the paths in the files to reflect the new directory layout - but might have missed some.

### CODE REVIEW

File: css/info.css [hunk 1]
```diff
@@ -1,296 +1,339 @@
...
-  background-color: #A8A9A7;
-  color: #002740;
-}
-input:active, input.active {
-  background-color: #8F908E;
-  border: solid #4F504F 1px;
-  border-bottom: solid #CFCFCE 1px;
-  border-right: solid #CFCFCE 1px;
-}
-*/
```
Comment (jrha) [inline css/info.css]: Might as well remove commented out lines.

File: css/info.css [hunk 2]
```diff
@@ -1,296 +1,339 @@
...
-  color: #f57900;
-}
-div#pakiti2-json span.moderate {
-  color: #3465a4;
-}
-div#pakiti2-json span.low, div#pakiti2-json span.standard {
-  color: #4e9a06;
-}
-
-/*PBS Nodetypes*/
```
Comment (jrha) [inline css/info.css]: It's not PBS anymore... "Batch Nodetypes"

File: css/info.css [hunk 3]
```diff
@@ -1,296 +1,339 @@
...
+  background-color: #A8A9A7;
+  color: #002740;
+}
+input:active, input.active {
+  background-color: #8F908E;
+  border: solid #4F504F 1px;
+  border-bottom: solid #CFCFCE 1px;
+  border-right: solid #CFCFCE 1px;
+}
+*/
```
Comment (jrha) [inline css/info.css:233]: Might as well remove commented out lines.

File: css/info.css [hunk 4]
```diff
@@ -1,296 +1,339 @@
...
+  color: #f57900;
+}
+div#pakiti2-json span.moderate {
+  color: #3465a4;
+}
+div#pakiti2-json span.low, div#pakiti2-json span.standard {
+  color: #4e9a06;
+}
+
+/*PBS Nodetypes*/
```
Comment (jrha) [inline css/info.css:290]: It's not PBS anymore... "Batch Nodetypes"

File: css/info.css [hunk 5]
```diff
@@ -1,296 +1,339 @@
...
+}
+
+span.rollup:hover {
+  color: rgb(52, 101, 164);
+  cursor: pointer;
+}
+
+div#node-magdb-history > table > tbody > tr > th, td {
+  font-size: 7pt;
+  /*border: thin solid rgb(225, 227, 222);*/
```
Comment (jrha) [inline css/info.css:326]: Remove?

**Comment by GooBall** [comment]
I haven't started looking at cleaning up the css yet. There is quite a lot that needs condensing and tweaking so I think I'll leave that until I have most of the files that will be serving the content uploaded and figured out - otherwise I'll be bouncing in and out of the css all the time.

**Comment by jrha** [comment]
Fairnuff.

**Pull request merged**